SpatialGDK/Documentation/content/porting-unreal-project-to-gdk.md

```diff
@@ -11,10 +11,14 @@
 Follow the [setup guide]({{urlRoot}}/setup-and-installing) to the end of **Setting up the Unreal GDK module and Starter Project** > **Cloning**.
 > To speed up build times, install IncrediBuild, FastBuild, or another build distributor.
 
+## Glossary
+`<ProjectRoot>` - The root folder of your Unreal project.  
+`<GameRoot>` - The folder containing your Unreal game files and source code. Usually called ‘Game’ e.g. `<ProjectRoot>/Game`  
```

Comment:
"The folder containing your game module". And remove the "Usually called Game" bit,that's not really true.

Reply:
`Source/ShooterGame` might be an example that is familiar.